config/settings.yml

```diff
@@ -3,3 +3,8 @@ home:
 help:
   link_help: "https://www.railstutorial.org/help"
   link_book: "https://www.railstutorial.org/book"
+about:
```

Comment:
khi nào nên dùng setting e ?

Reply:
Em nghĩ là dùng setting dùng để khai báo những giá trị chung dùng chung cho toàn bộ web để khi gọi chỉ cần gọi trong setting mà không phải tạo mới

Reply:
settings chứa các config của hệ thống.
vd độ dài của name, regex SDT. ...
e tìm hiểu thêm nhé

Reply:
Vâng ạ